csi/src/node.rs

```diff
@@ -553,32 +553,42 @@ impl node_server::Node for Node {
                 debug!("unstage: is iSCSI device path is {}", devpath);
                 (devpath, ShareProtocolNexus::NexusIscsi)
             }
-            // Must be nbd
-            _ => {
-                if let Some(nexus) =
-                    lookup_nexus(&self.socket, &volume_id).await?
-                {
-                    trace!(
-                        "unstage: nbd, device path is {}",
-                        nexus.device_path
-                    );
-
-                    if let Ok(url) = url::Url::parse(nexus.device_path.as_str())
+            _ => match nvmf_find(volume_id.as_str()) {
+                Some(devpath) => {
+                    debug!("unstage: is NVMF device path is {}", devpath);
```

Comment:
Don't need the "is" before NVMF.
Also, this is a debug message whereas the nbd equivalent message is a trace message. It might be better if these were consistent.

Reply:
https://github.com/openebs/Mayastor/pull/302/commits/103a309bb8f0d25e4b4cf62544a0d24d400cd998